src/test/java/com/orgofarmsgroup/util/AppConstants.java

```diff
@@ -0,0 +1,9 @@
+package com.orgofarmsgroup.util;
+
+public final class AppConstants {
+    public static class API_URI {
+        public static class USERS_API_URI {
+            public static final String USERS_ROOT = "/users";
+        }
+    }
```

Comment:
Can we follow better naming conventions for classes here?

Reply:
sure.